### PR TITLE
Change ssh example user from 'ubuntu to 'root'

### LINKED ISSUE #3510: Misleading instructions about accessing cloud provisioned hosts via SSH
The [instructions](https://flynn.io/docs/installation/cloud#ssh-access>) show using the 'ubuntu' user, but the key's actually set up for root (on DigitalOcean, at least).

### PR DESCRIPTION
**Pull request by danhawkes**
Fixes #3510

### CODE REVIEW

File: docs/content/installation/cloud.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ To login to a server provisioned by Flynn, you can specify the generated key and
 IP address of the server:
 
 ```text
-ssh -i ~/.flynn/installer/keys/flynn ubuntu@SERVER_IP
+ssh -i ~/.flynn/installer/keys/flynn root@SERVER_IP
```
Comment (lmars) [inline docs/content/installation/cloud.md]: The user varies based on the VM provider, we should leave the default as `ubuntu` but advise users that it differs by provider:

* DigitalOcean: `root`
* Azure: `flynn`

**Comment by lmars** [comment]
@danhawkes please prefix your commit message with `docs: ` and add a [DCO sign-off](https://github.com/flynn/flynn/blob/master/CONTRIBUTING.md#developers-certificate-of-origin).

**Comment by lmars** [comment]
@danhawkes do you intend to modify this patch?

**Comment by lmars** [comment]
@danhawkes I went ahead and did a new PR: #3526.

**Comment by danhawkes** [comment]
No - the new PR looks good, thanks.

**Pull request closed without merging**